### DISCUSSION TITLE
Libretube suddenly becomes unusable

### DISCUSSION THREAD

**Discussion by wxlqwplm** [community/community#61787] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Due to some reasons, since version 14.1, libretube has suddenly stopped working properly and needs to be started using VPN. It has been a month now and I hope it can be changed

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
